### PR TITLE
Tickets/sitcom 453

### PR DESCRIPTION
**Pull request by isotuela**
First iteration of the DIMM startup. I'd like to hear your comments before proceeding further.

### CODE REVIEW

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 1]
```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:7]: consider: 
.... "seeing" which is a metric for the amount of atmospheric turbulence effecting image quality.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 2]
```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:9]: Maybe add that the wedge results in a displacement of one image on the sensor?

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 3]
```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:10]: Consider:
... that in turn enables a prediction of the expected FWHM in long exposures.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 4]
```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
+
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures and a wedge prism at the entrance pupil of the telescope. 
+The rms of the fluctuations of the angular distance between the two images is proportional to the atmospheric optical turbulence, that in turn, allows to predict the seeing FWHM. 
+
+The main control software is divided into several programs. The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service,
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:12]: some weird line separations here.
new line after the period only.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 5]
```diff
@@ -4,11 +4,52 @@
...
+The main control software is divided into several programs. The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service, 
+mount and camera control, and the `preat` program that converts the output of the `dimm_tool` into seeing values.
+
+.. figure:: ./_static/program_structure.png
+    :name: DIMM program structure
+
+    Structure of the DIMM software
+
+
+Below is a list of some of the most useful DIMM manuals:
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:21]: Is there also a collection on docushare where all the DIMM documents are? If so, then it'd be useful to link that at the bottom... just so people know where to hunt for more stuff if needed.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 6]
```diff
@@ -4,11 +4,52 @@
...
+- A detailed description of the `preat` program and preat output: `Preat Description`_
+
+- Information on the different operating modes that might be useful for maintenance and troubleshooting: `DIMM soft description`_
+
+- Rubin's confluence page with information of the individual hardware components, history of Rubin's DIMM, SW installation instructions and other useful links: `DIMM Confluence Page`_
+
+- DIMM source code: `DIMM Source Code`_
+
+
+To start up the DIMM, please follow the procedure detailed in :ref:`DIMM Startup <DIMM-StartUp>`
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:48]: I wonder if this is better put after 1st sentence? 
I imagine most people on this page will be more interested on how to start it than the docs? 
Up to you....

File: Nighttime-Operations/Observing-Constraints/index.rst [hunk 7]
```diff
@@ -22,7 +22,7 @@ Weather Constraints
 
 This page defines the different types of operational constraints.
 
-Operating conditions used to define the Rubin construction project are found in `LTS-54 <ls.st/lts-54>`__.
+Operating conditions used to define the Rubin construction project are found in LTS-54.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observing-Constraints/index.rst:25]: why did you remove the link?
you might need to use `https://ls.st/lts-54` if you're getting a broken link.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 8]
```diff
@@ -4,11 +4,52 @@
...
+
+Below is a list of some of the most useful DIMM manuals:
+
+.. _DIMM General Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-general_spec-en_V1-1.pdf
+.. _DIMM Configuration: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-config_spec-en_V1-2.pdf
+.. _tt_master Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-master_spec-en_V1-4.pdf
+.. _tt_meteo Spec: https://github.com/lsst-ts/ts_dimm_app/blob/develop/doc/pdf/dev-dimm-tt-meteo_spec-en_V1-2.pdf
+.. _Preat Description: https://curl.sai.msu.ru/mass/download/doc/preat_desc2e.pdf 
+.. _DIMM Optical and mechanical design: https://curl.sai.msu.ru/mass/download/doc/description.pdf 
+.. _DIMM soft description: https://curl.sai.msu.ru/mass/download/doc/dimm_soft_description.pdf
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:29]: software? 
you'll need to fix where you reference it as well.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 9]
```diff
@@ -0,0 +1,151 @@
...
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
+
+.. _Dimm_StartUp-Automatic: 
+
+Starting up the DIMM in automatic mode
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:73]: Rightfully or wrongfully, we've been capitalizing the letters of all major words in titles (I think)

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+
+.. note:: Substitute the following parameters in the command below. `name` is your target name. `RA` is expressed in hours, float; `dec` in deg, float. Keep `mag` and `color` in 0.0, otherwise, it fails.  
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  python3 monitor_dimm_2.py manual --name HD4128 
+        --ra 0.7264917 --dec -17.9865833 --mag 0.0 --color 0.0 --sptype G9
+
+
+Make sure the DIMM CSC is enabled and data is flowing to the EFD.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:118]: Consider:
Verify the DIMM CSC is enabled and data is appearing in the EFD.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 11]
```diff
@@ -0,0 +1,151 @@
...
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  python3 monitor_dimm_2.py manual --name HD4128 
+        --ra 0.7264917 --dec -17.9865833 --mag 0.0 --color 0.0 --sptype G9
+
+
+Make sure the DIMM CSC is enabled and data is flowing to the EFD. 
+
+Once this mode is enabled, the DIMM will continue taking data until the target is not observable, but will run again on the following night. You can manually stop the DIMM from doing this by setting the ``ameba.mode=0``. See shutdown procedure.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:120]: new line after the period.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 12]
```diff
@@ -0,0 +1,151 @@
...
+
+and within this OpenTPL:
+
+.. code-block:: bash
+   
+        AUTH OK 0 0
+        1 set ameba.mode=1
+        
+
+Make sure the DIMM CSC is enabled and data is flowing to the EFD.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:95]: Consider:
Verify the DIMM CSC is enabled and data is appearing in the EFD.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 11 above)
Comment (patrickingraham) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:120]: Doesn't setting ameba.mode=0, does that put it in automatic mode? Please clarify.
I'm unclear if setting that mode means that it doesn't operate the next night, or if it goes to automatic?

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 13]
```diff
@@ -0,0 +1,151 @@
...
+.. _Dimm_StartUp-Manual:
+
+Starting the DIMM in manual mode
+====================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+In this mode of operations, the user needs to choose the target and run the ``monitor_dimm_2.py`` python script available in the dimm home directory:
+
+.. note:: Substitute the following parameters in the command below. `name` is your target name. `RA` is expressed in hours, float; `dec` in deg, float. Keep `mag` and `color` in 0.0, otherwise, it fails.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:110]: what about the spectral type?

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 14]
```diff
@@ -0,0 +1,151 @@
...
+Once this mode is enabled, the DIMM will continue taking data until the target is not observable, but will run again on the following night. You can manually stop the DIMM from doing this by setting the ``ameba.mode=0``. See shutdown procedure.
+
+To monitor the status of the DIMM program, see :ref:`monitor the DIMM <Dimm-Monitor>`.
+
+.. _Dimm-Monitor: 
+
+Monitor the DIMM program
+==========================
+
+There are two ways to monitor the DIMM hardware, program and outputs.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/Start-up.rst:129]: Is there a reason you don't mention any displays in LOVE?

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 15]
```diff
@@ -0,0 +1,151 @@
...
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-StartUp:
+
+###############################
+DIMM Startup
+###############################
+
+The DIMM software automatically performs seeing measurements from dusk until dawn. Limited attendance from user is required.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:11]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 16]
```diff
@@ -0,0 +1,151 @@
...
+.. _DIMM-StartUp:
+
+###############################
+DIMM Startup
+###############################
+
+The DIMM software automatically performs seeing measurements from dusk until dawn. Limited attendance from user is required. 
+
+The DIMM program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+At dawn, the telescope will park itself, and the dome will close. This cycle repeats the following evening, if left unattended, when the sun sets.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:14]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 17]
```diff
@@ -0,0 +1,151 @@
...
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will shut down the dome until conditions clear.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:17]: Considered 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 18]
```diff
@@ -0,0 +1,151 @@
...
+
+.. important::
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
+
+.. code-block:: bash
+
+        (base) you@Ios ~ % ssh dimm@139.229.191.103
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:36]: Yes, correct

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 19]
```diff
@@ -0,0 +1,151 @@
...
+ 
+
+Connect via telnet to the `tt-meteo`. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0`` Commands are prepended with a freely chosen command id (unsigned integer) Here increasing integers (1,2) were used.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:48]: Removed comment about 1,2.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 20]
```diff
@@ -0,0 +1,151 @@
...
+
+The following two commands can be written once the prompt appears. 
+
+.. code-block:: bash
+    
+        1 set sky.temp=-30
+    
+        2 set sky.status=0
+
+Quit telnet, pressing simultaneously ``Ctrl`` and  ``]``, write ``quit`` and click Enter.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:59]: Cheers

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 1 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:7]: Sounds better.

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 3 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:10]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 5 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:21]: Yes, working on that with David Cabrera.

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 6 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:48]: Yes, I think it's a great idea.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 10 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:118]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 11 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:120]: Agree. I'm writing the shutdown section more clearly.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 12 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:95]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 13 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:110]: Need to check this part in the source code. Just in case, add the spectral type, for the time being.

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 14 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:129]: No DIMM displays yet in LOVE.

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 4 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:12]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 21]
```diff
@@ -0,0 +1,151 @@
...
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
+
+.. code-block:: bash
+
+        (base) you@Ios ~ % ssh dimm@139.229.191.103
+
+The password is located on the observers 1Password Vault.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:38]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst [hunk 22]
```diff
@@ -0,0 +1,151 @@
...
+
+.. _DIMM-StartUp:
+
+###############################
+DIMM Startup
+###############################
+
+The DIMM software automatically performs seeing measurements from dusk until dawn. Limited attendance from user is required. 
+
+The DIMM program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:13]: Should I add the procedure here, or link it once it exists?

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 8 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:29]: :) 👍

File: Nighttime-Operations/Observing-Constraints/index.rst (hunk 7 above)
Comment (isotuela) [reply Nighttime-Operations/Observing-Constraints/index.rst:25]: 👍

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 9 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:73]: 💯

File: Nighttime-Operations/Observatory/DIMM/Start-up.rst (hunk 22 above)
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/Start-up.rst:13]: DIMM vendor program sounds good?

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 2 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst:9]: Have modified syntax. Let me know if it makes sense.

**Comment by patrickingraham** [comment]
The rebase made the linkage break on the comments, so I'll resolve the ones that are confusing and just add a comment to the new line (if required).

File: index.rst [hunk 23]
```diff
@@ -49,9 +49,11 @@ Safety: Quick Reference for Nighttime Personnel
 It does not replace the required safety training to be on the summit at night. 
 This includes:
 
-- Procedures on who (and how) to call when there is an issue
-- What to do in case of vehicle failure, power-outage, fire etc.
-- Emergency meeting locations
+- procedures on who (and how) to call when there is an issue
```
Comment (patrickingraham) [inline index.rst:52]: something weird happened with these bullets.
this is repeated by line 55.

Also, the first letters of a bullet should be capitalized.

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 24]
```diff
@@ -0,0 +1,195 @@
...
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+In this mode of operations, the user needs to choose the target and run the ``monitor_dimm_2.py`` python script available in the dimm home directory:
+
+.. note:: Keep `mag` and `color` in 0.0.  
+
+.. code-block:: bash
+
+        dimm@dimm:~$  python3 monitor_dimm_2.py manual --name {target name}
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:128]: please verify spectral type is an actual input!

**Review by patrickingraham — CHANGES_REQUESTED** [review]

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 25]
```diff
@@ -0,0 +1,195 @@
...
+The DIMM automatically performs seeing measurements from dusk until dawn. 
+
+Limited support from the user is expected. 
+
+The DIMM vendor program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+
+At dawn, the telescope will park itself, and the dome will close. This cycle repeats the following evening, unless :ref:`manually shutdown <Dimm-Shutdown>`, when the sun sets.
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will shut down the dome until conditions clear.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst:20]: 👍  It also parks the telescope and stops the seeing computing software until it all clears.

File: index.rst (hunk 23 above)
Comment (isotuela) [reply index.rst:52]: I don't believe I edited these.

Comment (isotuela) [reply index.rst:52]: In this branch, this looks different, but I have modified with your suggestion.

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 24 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst:127]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 26]
```diff
@@ -4,11 +4,59 @@
...
+The main services are `ameba` that is in charge of the observation, synchronizing dome, weather service, 
+mount and camera control, and the `preat` program that converts the output of the `dimm_tool` into seeing values.
+
+.. figure:: ./_static/program_structure.png
+    :name: DIMM program structure
+
+    Structure of the DIMM software
+
+
+To operate the DIMM (start up, monitor and shutdown), please follow :ref:`DIMM Startup, Shutdown and Monitor <DIMM-Ops>`
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 27]
```diff
@@ -0,0 +1,195 @@
...
+ 
+
+Connect via telnet to the ``tt-meteo``. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0``. Commands are prepended with a freely chosen command id (unsigned integer).
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 28]
```diff
@@ -0,0 +1,195 @@
...
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. 
+
+Limited support from the user is expected. 
+
+The DIMM vendor program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+
+At dawn, the telescope will park itself, and the dome will close. This cycle repeats the following evening, unless :ref:`manually shutdown <Dimm-Shutdown>`, when the sun sets.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 29]
```diff
@@ -0,0 +1,195 @@
...
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. 
+
+Limited support from the user is expected. 
+
+The DIMM vendor program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 30]
```diff
@@ -0,0 +1,195 @@
...
+
+.. _DIMM-Ops:
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. 
+
+Limited support from the user is expected.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 31]
```diff
@@ -0,0 +1,195 @@
...
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. 
+
+Limited support from the user is expected. 
+
+The DIMM vendor program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 32]
```diff
@@ -4,11 +4,51 @@
...
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
-    :glob:
+To operate the DIMM (start up, monitor and shutdown), please follow :ref:`DIMM Startup, Shutdown and Monitor <DIMM-Ops>`
 
-    *
\ No newline at end of file
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures at the entrance pupil of the telescope. 
+One of the apertures holds a wedge prism that results in the displacement of an image on the sensor, producing the two mentioned pair spots.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:12]: ... producing a pair of images.

it can be a pair, or two spots, but not a pair of two spots, which would be four spots :)

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 33]
```diff
@@ -4,11 +4,51 @@
...
 
-    *
\ No newline at end of file
+The DIMM principle is to produce "twin images" of a bright star by a mask with two apertures at the entrance pupil of the telescope. 
+One of the apertures holds a wedge prism that results in the displacement of an image on the sensor, producing the two mentioned pair spots. 
+
+.. figure:: ./_static/DIMM_image.png
+    :name: DIMM image
+
+    The "twin images" of a star with the two-subaperture with one wedge mask system
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:17]: The "twin images" of a star imaged on the DIMM focal plane.

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 34]
```diff
@@ -0,0 +1,194 @@
...
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-Ops:
+
+######################################
+DIMM Startup, Shutdown and Monitoring
+######################################
+
+The DIMM automatically performs seeing measurements from dusk until dawn. Limited support from the user is expected.
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:11]: The "Limited support from the user is expected" sentence should be on a new line.

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 35]
```diff
@@ -4,11 +4,51 @@
...
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", which is a metric for the amount of atmospheric turbulence affecting image quality. 
 
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
-    :glob:
+To operate the DIMM (start up, monitor and shutdown), please follow :ref:`DIMM Startup, Shutdown and Monitor <DIMM-Ops>`
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:9]: Monitoring

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 34 above)
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:11]: "Limited support from the user is expected." -- belongs on it's own line since it's a separate sentence.

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 36]
```diff
@@ -0,0 +1,194 @@
...
+
+The vendor provided DIMM software will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+At dawn, the telescope will park itself, and the dome will close. 
+This cycle repeats the following evening, unless :ref:`manually shutdown <Dimm-Shutdown>`, when the sun sets.
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will close the dome until conditions clear. 
+
+Currently, there is no incoming Cerro Pachón weather information feeding the `tt-meteo` program. 
+
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:21]: remove this empty line or it puts the next sentence in a new paragraph

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 37]
```diff
@@ -0,0 +1,194 @@
...
+Every time the DIMM is restarted, the weather startup values are set so the operations are disabled for safety reasons. 
+
+To allow operations, mock weather data into the `tt-weather` service must be entered in the following manner:
+
+.. important::
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:37]: Consider:

When connected to the LSST WAP network, open a terminal and connect to the DIMM VM using:

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 38]
```diff
@@ -0,0 +1,194 @@
...
+
+.. figure:: ./_static/tt-meteo.png
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. 
+
+Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:79]: this should be on line 78 with the rest of the sentence

File: Nighttime-Operations/Observatory/DIMM/ops.rst [hunk 39]
```diff
@@ -0,0 +1,194 @@
...
+To monitor the status of the DIMM program, see :ref:`monitor the DIMM <Dimm-Monitor>`.
+
+.. _Dimm_StartUp-Manual:
+
+Starting up the DIMM in Manual Mode
+====================================
+
+Make sure you have run and read the :ref:`prerequisites <Dimm_StartUp-Prerequisites>` regarding the weather data. 
+
+In this mode of operations, the user needs to choose the target and run the ``monitor_dimm_2.py`` python script available in the dimm home directory:
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/ops.rst:121]: do you have to turn off automatic mode first? It would be helpful to add 1 short sentence saying if that is or is not required.

**Review by patrickingraham — CHANGES_REQUESTED** [review]
Much better. Thanks for making the changes.
I made a few suggestions, mostly grammar.
If you could fix those then squash the commits that would be awesome and we can merge this!
I'm happy to walk you through the git stuff if you are uncertain, don't hesitate to ask.

**Review by patrickingraham — CHANGES_REQUESTED** [review]
hmm... i somehow started two reviews... putting these comments too although they're probably repeats!

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 39 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst:121]: Right. 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 38 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 37 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 34 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: Repeated comment. 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 35 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/ops.rst (hunk 34 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/ops.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 33 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst (hunk 32 above)
Comment (isotuela) [reply Nighttime-Operations/Observatory/DIMM/index.rst]: 👍

File: Nighttime-Operations/Observatory/DIMM/index.rst [hunk 40]
```diff
@@ -4,11 +4,57 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", which is a metric for the amount of atmospheric turbulence affecting image quality. 
+
+To operate the DIMM (start up, monitor and shutdown), please follow :ref:`DIMM Startup, Shutdown and Monitoring <DIMM-Ops>`
 
 .. toctree::
```
Comment (patrickingraham) [inline Nighttime-Operations/Observatory/DIMM/index.rst:11]: I don't think this is necessary as you link the only content just above.

**Review by patrickingraham — APPROVED** [review]
Thanks for making these changes!
